.gitignore

```diff
@@ -5,6 +5,7 @@ src/dogecoin
 src/dogecoind
 src/dogecoin-cli
 src/dogecoin-tx
+src/bench
```

Comment:
```suggestion
```
Lets cut these back out, and we can the squash the lot in GitHub and commit them later.